package.json

```diff
@@ -10,23 +10,26 @@
   "scripts": {
     "postinstall": "npm ls --depth=0",
     "start": "gulp dev",
-    "heroku": "gulp copy-assets && gulp sassdoc && node app/start.js",
+    "heroku": "npm run build:assets && node app/start.js",
     "build-release": "./bin/build-release.sh",
     "publish-release": "./bin/publish-release.sh",
     "pre-release": "./bin/pre-release.sh",
+    "build:assets": "gulp copy-assets && gulp sassdoc",
     "build:package": "gulp build:package --destination 'package' && npm run test:build:package",
     "build:dist": "gulp build:dist --destination 'dist' && npm run test:build:dist",
-    "test": "gulp copy-assets && jest --testPathIgnorePatterns=after-*",
-    "test:build:package": "jest tasks/gulp/__tests__/after-build-package.test.js",
-    "test:build:dist": "jest tasks/gulp/__tests__/after-build-dist.test.js",
+    "pretest": "npm run build:assets",
+    "test": "jest --color --ignoreProjects \"Gulp tasks\" --maxWorkers=50%",
+    "test:build:assets": "npx jest --selectProjects \"Gulp tasks\" --testMatch \"**/*components*\"",
```

Comment:
Is this intended to run `check-individual-components-compile.test.js`?

I had no idea that test file existed…

1. I think it's running on the contents of the `src` directory, so don't think it makes sense to run it after `build:assets` as that sort of suggests it's running tests against the copied files in the `public` directory.
2. I think the test basically duplicates tests we already have in `components/all.test.js`:
   https://github.com/alphagov/govuk-frontend/blob/a1a1944b174910dc76032bc9e9318a7cfc0173f6/src/govuk/components/all.test.js#L40-L44

Reply:
If both of those are correct, I think we should delete `check-individual-components-compile.test.js` (because it's covered by the tests in `src/govuk/components/all.test.js`) and remove the `test:build:assets` script.

Reply:
Yep that was the intention, agreed. I'll remove 👍

Reply:
Removed it now, well spotted. Don't need to call `renderSass()` on every component twice 😊